README.md

```diff
@@ -46,41 +46,29 @@ When we make an official release, we will create a unique git tag containing the
 
 2. Clone our Azure SDK repository, optionally using the desired version tag.
 
-    ```bash
-    git clone https://github.com/Azure/azure-sdk-for-c
-    ```
+        git clone https://github.com/Azure/azure-sdk-for-c
 
-    ```bash
-    git checkout <tag_name>
-    ```
+        git checkout <tag_name>

```

Comment:
nit: Is there a way to retain the new line between these two commands since the caller doesn't need to use both (just one)? Maybe we need another new line.

Before (markdown rendering from master):
![image](https://user-images.githubusercontent.com/6527137/81746904-12b56000-945c-11ea-88e2-96749e1628b4.png)

After:
![image](https://user-images.githubusercontent.com/6527137/81746829-f7e2eb80-945b-11ea-8f8d-a033c1bdf50a.png)

Reply:
Sadly this is a problem with having 2 different documentation rendering approaches. GitHub generates one set of HTML outputs, Doxygen generates another HTML based on design decisions appropriate to Doxygen (http://www.doxygen.nl/manual/markdown.html#markdown_dox)